packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/Choice.spec.js

```diff
@@ -60,5 +67,13 @@ describe('Component > Choice', function () {
     expect(wrapper.find(Questions)).to.have.lengthOf(1)
   })
 
-  // TODO: add tests for Identify button disabled prop once state passed down as prop
+  it('should call onCancel with "Not this" button click', function () {
+    wrapper.find(Button).simulate('click')
+    expect(onCancelSpy).to.have.been.calledOnce()
```

Comment:
I don't think so, as no args are passed in `onCancel`, instead relying on the `handleCancel` in `SurveyTask` to call the more specific handlers with args up to the container. Which makes me wonder if `handleCancel` is in the right place though, maybe should be moved from SurveyTask to SurveyTaskContainer?

Reply:
Ok, well, the description is confusing because I would expect onCancel to be called with "Not this" here based on that. Perhaps reword the description to better reflect what's being tested?

Reply:
Not sure why I used "with", definitely confusing, edited for clarity.